features/works/work_related.feature

```diff
@@ -583,3 +583,14 @@ Scenario: When a user is notified that a co-authored work has been inspired by a
       And I should not see "A work in an unrevealed collection"
       And I should not see "Worldbuilding Translated by translator"
       And I should not see "From English to Deutsch"
+
+  Scenario: A work in a hidden collection should not be revealed by posting a related work
```

Comment:
Could we rephrase this for clarity? Right now, it sounds like we're checking that the work itself isn't getting revealed. "Scenario: A work that is inspired by a work in an unrevealed collection does not list the title or creator of the unrevealed work in its notes" or something similar might be clearer.